examples/warm_up_example/warmed_up_fenda/client.py

```diff
@@ -0,0 +1,134 @@
+            pretrained_model_path=Path(os.path.join(pretrained_model_dir, pretrained_model_name)),
+            weights_mapping_path=weights_mapping_path,
+        )
+
+    def get_data_loaders(self, config: Config) -> Tuple[DataLoader, DataLoader]:
+        sampler = DirichletLabelBasedSampler(list(range(10)), sample_percentage=0.75, beta=1)
+        batch_size = self.narrow_config_type(config, "batch_size", int)
+        train_loader, val_loader, _ = load_mnist_data(self.data_path, batch_size, sampler)
+        return train_loader, val_loader
+
+    def get_model(self, config: Config) -> nn.Module:
+
+        # Load the pretrained model
+        model: nn.Module = self.warmed_up_module.load_from_pretrained(
+            FendaModel(LocalCnn(), GlobalCnn(), FendaClassifier(FendaJoinMode.CONCATENATE))
+        ).to(self.device)
+
+        # To not overwrite the pretrained model with server model weights
+        self.model_weights_initialized = True
+        return model
+
+    def get_optimizer(self, config: Config) -> Optimizer:
+        return torch.optim.SGD(self.model.parameters(), lr=0.001, momentum=0.9)
+
+    def get_criterion(self, config: Config) -> _Loss:
+        return torch.nn.CrossEntropyLoss()
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="FL Client Main")
+    parser.add_argument("--dataset_path", action="store", type=str, help="Path to the local dataset")
+    parser.add_argument(
+        "--server_address",
+        action="store",
+        type=str,
+        help="Server Address for the clients to communicate with the server through",
+        default="0.0.0.0:8080",
+    )
+    parser.add_argument(
+        "--seed",
+        action="store",
+        type=int,
+        help="Seed for the random number generator",
+        required=False,
+    )
+    parser.add_argument(
+        "--pretrained_model_dir",
+        action="store",
+        type=str,
+        help="Path to the pretrained model",
+        required=False,
+    )
+    parser.add_argument(
+        "--client_number",
+        action="store",
+        type=int,
+        help="Number of the client for the loading of the pretrained model",
+        required=True,
+    )
+    parser.add_argument(
+        "--weights_mapping_path",
+        action="store",
+        type=str,
+        help="Path to the weights mapping file",
+        required=False,
+    )
+    args = parser.parse_args()
+
+    DEVICE = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+    data_path = Path(args.dataset_path)
+    log(INFO, f"Device to be used: {DEVICE}")
+    log(INFO, f"Server Address: {args.server_address}")
+
+    # Set the random seed for reproducibility
+    set_all_random_seeds(args.seed)
+
+    # Start the client
+    client = MnistFendaClient(
+        data_path,
+        [Accuracy("accuracy")],
+        DEVICE,
+        args.client_number,
+        Path(args.pretrained_model_dir),
+        Path(args.weights_mapping_path),
```

Comment:
Similar to the FedProx example, I think we can make a few small refactors to these arguments to make them a bit more graceful. Let me know if you have any objections.